packaging/python/test_wheels.sh

```diff
@@ -260,7 +260,7 @@ $python_exe -m pip install --upgrade pip
 # install numpy, pytest and neuron
 # we install setuptools because since python 3.12 it is no more installed
 # by default
-$python_exe -m pip install "numpy<2" pytest setuptools
+$python_exe -m pip install "numpy" pytest setuptools
```

Comment:
See above, do we really need to install `numpy`? Should that not be a dependency of the Neuron wheel in the next line?